website/docs/verifiable-document/did/revoking-document.md

```diff
@@ -3,4 +3,52 @@ title: Revoking Documents
 sidebar_label: Revoking Documents
 ---
 
-As of today, it's not possible to revoke a signed document but we are working on it. You can follow [our progress](https://github.com/Open-Attestation/adr/blob/master/issuing_using_did.md#for-documents-that-are-signed-directly).
+As of today *180221*, its possible to revoke a signed document only if you fulfill certain **prerequisites**. Any future progress can be followed [here](https://github.com/Open-Attestation/adr/blob/master/issuing_using_did.md#for-documents-that-are-signed-directly).
+
+## Prerequisites:
+- an `ethr` DID (if you've been following the above steps, you should have one already)
```

Comment:
We also need a document store deployed, as this part doesn't explain how to do it

I would suggest to link to deploy a document store

Reply:
ah yea okay